.github/workflows/post-apiview.yml

```diff
@@ -0,0 +1,27 @@
+name: After APIView
+
+on:
+  check_suite:
+    types: [completed]
+    
+permissions:
+  pull-requests: write
+  contents: read
+
+jobs:
+  event-handler:
+    name: After APIView
+    runs-on: ubuntu-24.04
+    if: github.event.check_suite.app.name == 'Azure Pipelines'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          sparse-checkout: 'eng/common'
+          fetch-depth: 0
+    
+      - name: Create APIView Comment on PR
+        run: |
+          . ../../../eng/common/scripts/helpers/ApiView-Helpers.ps1
+          Set-ApiViewCommentForRelatedIssues -HeadCommitish ${{ github.event.check_suite.head_sha }} -AuthToken ${{ secrets.GITHUB_TOKEN }}
+        shell: pwsh
```

Comment:
Our files should end with a trailing newline.  I keep adding it back, but you keep deleting it again.